bigtable/conformance_test.sh

```diff
@@ -50,11 +50,11 @@ trap cleanup EXIT
 
 # Run the conformance tests
 cd $conformanceTestsHome
-# Tests in https://github.com/googleapis/cloud-bigtable-clients-test/tree/main/tests can only be run on go1.22.7
-go install golang.org/dl/go1.22.7@latest
-go1.22.7 download
+# Tests in https://github.com/googleapis/cloud-bigtable-clients-test/tree/main/tests can only be run on go1.24.0
+go install golang.org/dl/go1.24.0@latest
+go1.24.0 download
```

Comment:
The Go version used was previously the "earliest" (1.22.7). Do you want to keep it at the new "earliest" (1.23.0)?